chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -71,6 +71,13 @@
 
   include_recipe "crowbar::prepare-upgrade-scripts"
 
+  bash "set pre-upgrade node attribute"
```

Comment:
Since this is a one-liner, why not use the [`execute` resource](https://docs.chef.io/resource_execute.html) ?